app/controllers/admin/data_exports_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::DataExportsController < Admin::AdminController
+  before_action :set_export_class, only: :new
+
+  def new
+    @data_export = @export_class.new
+  end
+
+  private
+
+  def set_export_class
+    @export_class = DATA_EXPORT_TYPES[params[:export_type]].constantize
+  end
+end
```

Comment:
cool!

Reply:
If it works! This can't really be tested properly until we start subclassing.